config/credentials.yml.enc.example

```diff
@@ -0,0 +1 @@
+vDBb70N1EOfXSJQkulh6YoytzJ6ZxZil5emyq56shoaVVbb/0LZ7/SuaVUuPnfDDaIuhnxBe2ntnXC9evQYBcVJEEP0EIOGt54+LEmWEOw6PS3LYpZ6ohq7G7G/8uiIBf8xzeskbUyI/a0dKaP1ms5rewUQaVYrUTyYMEjF3KNoWTzobgZz3N0hhisfzmjqNBftL7anqFuWNOgUtCNFf/AYIVathJeJrs+IZXRl5o069usUR8gCvh4tKjJ5VIf2YXuTdtAyxB1nO2Z1zq5vqW7K+P4HxUe9OLdM1oYBO4UhcVIfkhEN5f+ymRQuKFyI99cuQuJjdIjcNPboGkL1Zw8iFKmYMOR98BIHN38Py1luLupKlA1P2okC+skkiHYEDTyuBAr7nUjHE+Hm/jqQqMR8x+TKvHIYHzniu--GFv0RPL/l9APij5o--H5qinJy9X8xzf5IsXWHZqQ==
```

Comment:
you have to create credentials.yml.enc.example and push to git
this file you add to gitignore, same your database.yml

Reply:
I did this earlier, this example file is the file I created earlier before deleting credentials.yml.enc. Do I need to do this again for example file?